.github/workflows/test.yml

```diff
@@ -16,6 +16,7 @@ on:
     - "**.md"
 env:
   LIMACTL_CREATE_ARGS: ""
+  LIMA_GRPC_PORT_FORWARDER: true
```

Comment:
Can we just enable this by default?

Reply:
yes we can. I did this just to be on safer side and not break any exiting feature.

If you feel we can make this default let me know i will update it

Reply:
I guess this can be enabled by default

Reply:
Done.